android/app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,6 @@
 <resources>
-  <string name="app_name">Expo Go</string>
+  <string name="versioned_app_name">Expo Go</string>
+  <string name="unversioned_app_name">Expo Go (unversioned)</string>
```

Comment:
Isn't it too long for Android's UI? On iOS the app name should be up to 12-13 chars to not get ellipsized on the home screen — that's why I used `Expo Go` too. I would rather change the icon to differentiate these two flavors 😉 (I already prepared a new icon for iOS, so I think we can update them as the next step)